docs/openstack_dataplanenode.md

```diff
@@ -78,6 +79,20 @@ NodeSection is a specification of the node attributes
 | ansibleVars | AnsibleVars for configuring ansible | string | false |
 | ansibleSSHPrivateKeySecret | AnsibleSSHPrivateKeySecret Private SSH Key secret containing private SSH key for connecting to node. Must be of the form: Secret.data.ssh-privatekey: <base64 encoded private key contents> https://kubernetes.io/docs/concepts/configuration/secret/#ssh-authentication-secrets | string | true |
 | extraMounts | ExtraMounts containing files which can be mounted into an Ansible Execution Pod | []storage.VolMounts | true |
+| nova | NovaTemplate specifies the parameters for the compute service deployment on the EDPM node.If it is specified both in OpenstackDataPlaneRole and the OpenstackDataPlaneNode for the same EDPM node then the configuration in OpenstackDataPlaneNode will be used and the configuration in the OpenstackDataPlaneRole will be ignored. If this is defined in neither then compute service(s) will not be deployed on the EDPM node. | *[NovaTemplate](#novatemplate) | true |
+
+[Back to Custom Resources](#custom-resources)
+
+#### NovaTemplate
+
+NovaTemplate specifies the parameters for the compute service deployment on the EDPM node.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| cellName | CellName is the name of nova cell the compute(s) should be connected to | string | true |
+| novaInstance | NovaInstance is the name of the Nova CR that represents the deployment the compute(s) belongs to | string | true |
```

Comment:
"the name of the Nova CR that represents the deployment the compute(s) belongs to" 
I have some idea of what this means but I'm not certain. If you can provide an example it would make it more clear.

I would add an extra paragraph, outside of this table, which shows a deployment ready to get an EDPM compute node. An example command would then retrieve a NovaCR value so the reader knows for sure what you are referring to.

Reply:
I added the command to the doc to query the names of the Nova CRs in the system.
If we always deploy via openstack-operator then the default value "nova" is good as openstack-operator hardcodes that as a name of the Nova CR it creates.